tensorflow_addons/custom_ops/image/cc/kernels/resampler_ops_gpu.cu.cc

```diff
@@ -0,0 +1,283 @@
+// Copyright 2016 The Sonnet Authors. All Rights Reserved.
```

Comment:
Hi @WindQAQ, I'd like to know if we need to edit the first line here (license). What is your suggestion?